robin88322/docker/example5/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu
+
+RUN apt-get update && apt-get install apache2 -y && service apache2 start
```

Comment:
Command 'service apache2 start' is not necessary here (in the 'docker build' commands chain) as service will be started at the container start stage.